content/en/docs/tasks/configure-pod-container/share-process-namespace.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: Share Process Namespace between Containers in a Pod
-min-kubernetes-server-version: v1.10
```

Comment:
If we remove this line, we should also remove the `{{< version-check >}}` shortcode from prerequisites @tengqm.

Reply:
OK, let's remove it then.